test/ref/resize.cpp

```diff
@@ -421,7 +421,7 @@ TEST_CASE(resize_fail_test_4)
                                            {"coordinate_transformation_mode", "invalid"}}),
                         {a0});
     p.compile(migraphx::make_target("ref"));
-    EXPECT(test::throws([&] { p.eval({}).back(); }));
+    EXPECT(test::throws([&] { std::ignore = p.eval({}); }));
```

Comment:
We shouldn't need the `std::ignore` as `p.eval` does not have the `no_discard` attribute.